Data/src/main/java/org/tribuo/data/columnar/processors/response/BinaryResponseProcessor.java

```diff
@@ -47,6 +52,36 @@
     @Config(description="The negative response to emit.")
     private String negativeName = "0";
 
+    @Config(description = "A list of field names to read, you should use only one of this or fieldName.")
+    private List<String> fieldNames;
+
+    @Config(description = "A list of strings that trigger positive responses; it should be the same length as fieldNames or empty")
+    private List<String> positiveResponses;
+
+    @Config(description = "Whether to display field names as part of the generated label, defaults to false")
+    private boolean displayField = false;
+
+
+    @Override
+    public void postConfig() {
+        if (fieldName != null && fieldNames != null) { // we can only have one path
+            throw new PropertyException("fieldName, FieldNames", "only one of fieldName or fieldNames can be populated");
```

Comment:
The first argument to the three string constructor for `PropertyException` is supposed to be the name of the object in the config file. In practice I don't bother putting in the `ConfigurableName` annotation just to get that information, though if I'd thought more clearly about it I guess it could have been an argument to `postConfig`. Either way, you should probably set it to the empty string to mirror the rest of the places in Tribuo where we don't have this information and throw `PropertyException`.